DEVELOPER.md

```diff
@@ -37,7 +37,7 @@ and the most important variables, run `make help`, e.g:
 VARIABLES:
   APPS          = config configsubscription firehose forwarder logwriter metricstream stack
   AWS_REGION    = us-west-2
-  GO_BINS       = forwarder subscriber
+  GO_BINS       = forwarder subscriber metricsrecorder
```

Comment:
what metrics is this new app recording? really it updates the existing metrics stream right? maybe we could call it `metricstreamupdater` or `metricsconfigurator`